integrations/extensions/docs/elasticsearch-install-and-setup/text_embedding_deploy_and_use.md

```diff
@@ -219,4 +223,13 @@ curl -X GET "${ES_URL}/${ES_EMBEDDING_INDEX_NAME}/_search" \
     "text"
   ]
 }'
-```
\ No newline at end of file
+```
+
+## What's next?
+Now that you've successfully deployed your text embedding model in Elasticsearch, here are the recommended next steps:
+### 1. Configure your assistant for document search using a custom Elasticsearch extension
+Use the vector representations generated by your deployed model to enhance document search in your assistant. Follow the instructions provided [here](../../starter-kits/elasticsearch/README.md) to integrate Elasticsearch as a custom extension into your assistant.
+### 2. Use search results from Elasticsearch as input context to an LLM
+Follow the example provided [here](../../starter-kits/language-model-conversational-search/README.md) to leverage the search results from Elasticsearch as input context to a watsonx large language model (LLM) to generate contextually relevant responses to user queries.
+### 3. Use the built-in Elasticsearch extension for document search answer generation in your assistant
+Follow the instructions [here](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-search-elasticsearch-add) to set up the Elasticsearch search integration. Then enable dense vector search for your search extension on Watson Asistant by following the guide [here](README.md#2-set-up-the-built-in-elasticsearch-extension).
```

Comment:
The link should work once this PR is merged. or you could rebase this PR to get the latest from the master branch since the other readme PR has been merged.

Reply:
Thanks, Zach! I rebased master. Should be good now.